### PR TITLE
chore: bump dep versions

### PR DESCRIPTION
**Pull request by JFrankfurt**

### CODE REVIEW

**Review by zhongeric — APPROVED** [review]
nice, just curious why we need to bump sdk core?

File: yarn.lock [hunk 1]
```diff
@@ -5085,9 +5085,9 @@
     "@uniswap/v3-sdk" "^3.8.3"
 
 "@uniswap/sdk-core@^3.0.0-alpha.3", "@uniswap/sdk-core@^3.0.1", "@uniswap/sdk-core@^3.1.0", "@uniswap/sdk-core@^3.2.0":
-  version "3.2.0"
-  resolved "https://registry.yarnpkg.com/@uniswap/sdk-core/-/sdk-core-3.2.0.tgz#cabee6b21469d2e8d41c367503cfd961dddc8305"
-  integrity sha512-KXCIDNRhKF9DeDBY2j7SiAlD56ZqD/T6his2m9y8ZCMdObLBVt/oCt4kaQGrWUjJuf2Flr5dryHMmMQBXqA9pA==
+  version "3.2.2"
```
Comment (zzmp) [inline yarn.lock:5088]: looks like this snuck in there - it shouldn't be touched based on the package.json changes

**Review by zzmp — CHANGES_REQUESTED** [review]
please include in PR description _why_ we need to bump these version

**Comment by JFrankfurt** [comment]
@zzmp

Eric Zhong
  [2 hours ago](https://uniswapteam.slack.com/archives/C04QET5UFEJ/p1678485052049259)
We just released smart order router 3.6.1 to support the new bsc deployment addresses! 
[@Jordan Frankfurt](https://uniswapteam.slack.com/team/U01KN7JV9U7)
 feel free to bump in interface to support client side routing

**Review by zzmp — APPROVED** [review]

**Pull request merged**